module/sheets/WitcherActorSheet.js

```diff
@@ -150,8 +150,10 @@ export default class WitcherActorSheet extends ActorSheet {
         if (this[i]["system"][prop]) {
           total += Number(this[i]["system"][prop])
         }
-        else if (this[i]["system"]["system"][prop]) {
-          total += Number(this[i]["system"]["system"][prop])
+        else if (this[i]["system"]["system"]) {
+          if(this[i]["system"]["system"][prop]) {
```

Comment:
wouldn't it be better to do

`
this[i].system?.system?.prop
`

Reply:
yes, I think a better way to do that. I'll give it a try when i have a chance to.

Sorry, I'm not that used to javascript and didn't know you could do optional access this way.

Anyway I'll give it a try and update the PR accordingly